src/forms.ts

```diff
@@ -81,6 +116,20 @@ export class Forms {
   static Init(){
     document.addEventListener("DOMContentLoaded", () => {
 
+      document.addEventListener('change', (e: KeyboardEvent) => {
```

Comment:
yeah kinda, here I find kinda wierd checking for `input` element inside the function and outside of it. But, to be frankly speaking, the old jQuery code only checks for the length and applies the validate_field immediately.

Referencing here from v1.0.0:
````
$(document).on('change', input_selector, function() {
      if (this.value.length !== 0 || $(this).attr('placeholder') !== null) {
        $(this)
          .siblings('label')
          .addClass('active');
      }
      M.validate_field($(this));
    });
````

Since the text are now being handled by css, I think we can drop the length check.